tools/tox.ini

```diff
@@ -21,3 +21,8 @@ commands = flake8 --append-config={toxinidir}/py27-flake8.ini {posargs}
 [testenv:py36-flake8]
 deps = -rrequirements_flake8.txt
 commands = flake8 --append-config={toxinidir}/py36-flake8.ini {posargs}
+
+[testenv:mypy]
+basepython = python3
+deps = mypy
```

Comment:
Let's pin `mypy` version (and probably all deps?) in a `requirements_mypy.txt` (and add that to the pyup config), because we don't want this to start failing suddenly.